.changeset/tall-pumpkins-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `--no-deprecation` to suppress rule deprecation warnings
```

Comment:
[suggest] How about adding a few words to help people distinguish which flag of Node.js or Stylelint?

```suggestion
Fixed: Node.js `--no-deprecation` flag to suppress rule deprecation warnings
```

Reply:
```suggestion
Fixed: honour Node.js `--no-deprecation` flag for rule deprecation warnings
```